src/subdataframe/subdataframe.jl

```diff
@@ -51,7 +51,7 @@ immutable SubDataFrame{T <: AbstractVector{Int}} <: AbstractDataFrame
     parent::DataFrame
     rows::T # maps from subdf row indexes to parent row indexes
 
-    function SubDataFrame(parent::DataFrame, rows::T)
+    @compat function SubDataFrame{T}(parent::DataFrame, rows::T) where T
```

Comment:
Nevermind, I just found the notes in https://github.com/JuliaLang/julia/pull/20308/files.